azure-cli/webapp/New-WebApp.ps1

```diff
@@ -0,0 +1,39 @@
+function New-WebApp {
+  param (
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [Alias("Name", "AppName")]
+    [string]
+    $WebAppName,
+
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [Alias("Plan")]
+    [string]
+    $AppServicePlanId,
+
+    [Parameter(Mandatory = $true)]
+    [ValidateNotNullOrEmpty()]
+    [string]
+    $ResourceGroupName,
+
+    [Parameter(Mandatory = $false)]
```

Comment:
We should be able to generate a validationSet for this property:

az webapp list-runtimes
```

[
  "aspnet|V4.8",
  "aspnet|V3.5",
  "DOTNETCORE|2.1",
  "DOTNETCORE|3.1",
  "DOTNET|5.0",
  "DOTNET|6.0",
  "node|10.6",
  "node|10.10",
  "node|10.14",
  "node|12-lts",
  "node|14-lts",
  "php|7.4",
  "java|1.8|Tomcat|7.0",
  "java|1.8|Tomcat|8.5",
  "java|1.8|Tomcat|9.0",
  "java|1.8|Java SE|8",
  "java|7|Tomcat|7.0",
  "java|7|Tomcat|8.5",
  "java|7|Tomcat|9.0",
  "java|7|Java SE|8",
  "java|11|Tomcat|7.0",
  "java|11|Tomcat|8.5",
  "java|11|Tomcat|9.0",
  "java|11|Java SE|8"
]
```